.circleci/config.yml

```diff
@@ -39,60 +39,81 @@ jobs:
   docs:
     <<: *common
     docker:
-      - image: cimg/python:3.7
+      - image: cimg/python:3.8
```

Comment:
Our standard is to build docs with the lowest supported py version, but 3.7 is deprecating in a couple months, so what the heck?